app/views/home/index.html.slim

```diff
@@ -28,7 +28,7 @@
     h2.govuk-heading-m = t(".jobseeker_section.title")
     - if jobseeker_signed_in?
       p.govuk-body = t(".jobseeker_section.signed_in.description")
-      = govuk_link_to t("buttons.view_account"), jobseeker_root_path, class: "govuk-link--no-visited-state"
+      p = govuk_link_to t("buttons.view_account"), jobseeker_root_path, class: "govuk-link--no-visited-state"
```

Comment:
did you mean to put this in a p tag? cant you add an inline govuk class like you have below?